### PR TITLE
add motivation and reorder tutorials page

### PR DESCRIPTION
**Pull request by ErlendHov82**

### CODE REVIEW

File: document-classification/content.md [hunk 1]
```diff
@@ -292,12 +296,6 @@ mflux_ai.init("your_project_token_goes_here")
 mlflow.keras.autolog()
 ```
 
-`mlflow.keras.autolog()` performs one HTTP request against the MLflow server after each epoch,
```
Comment (iver56) [inline document-classification/content.md:295]: Uh-oh, it looks like you are accidentally reverting a change I did earlier

File: log-batch/content.md [hunk 2]
```diff
@@ -82,9 +79,3 @@ with mlflow.start_run() as active_run:
 
     mlflow_client.log_batch(run_id=active_run.info.run_id, metrics=all_metrics)
 ```
-
-Note that there is a limit on the number of metrics that you can log in a single `log_batch`
-call. This limit is typically 1000. If you exceed the limit, you'll get an error like this:
```
Comment (iver56) [inline log-batch/content.md:87]: Uh-oh, it looks like you are accidentally reverting a change I did earlier

**Review by iver56 — CHANGES_REQUESTED** [review]

File: boston-crime-forecasting/content.md [hunk 3]
```diff
@@ -1,4 +1,10 @@
 # Analysis and Forecasting on Boston Crime Data
+This tutorial shows something close to a real data science project. It is more complicated than most other tutorials on
```
Comment (ojsvendsli) [inline boston-crime-forecasting/content.md:2]: This tutorial shows a small data science project.

File: document-classification/content.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+By classifying documents, we are aiming to assign one or more classes or categories to a document, making it easier to
```
Comment (ojsvendsli) [inline document-classification/content.md:1]: By classifying documents, we are assigning one or more classes or categories to a document

**Comment by iver56** [comment]
Closed because https://github.com/AIAScience/mflux-ai-tutorials/pull/34 took over

**Pull request closed without merging**